bson/src/main/org/bson/BasicBSONDecoder.java

```diff
@@ -22,11 +22,47 @@
 import java.io.InputStream;
 import java.nio.ByteBuffer;
 
+import static org.bson.assertions.Assertions.notNull;
+
 /**
  * Basic implementation of BSONDecoder interface that creates BasicBSONObject instances
  */
 public class BasicBSONDecoder implements BSONDecoder {
 
+    /**
+     * Sets the default {@link UuidRepresentation} to use when decoding BSON binary values.
```

Comment:
Perhaps add warning that this globally sets the default for all users of `BasicBSONDecoder` within the JVM.

Reply:
Done